docs/General/buildRPC.md

```diff
@@ -13,117 +7,41 @@ The Remote Provisioning Client (RPC) communicates with the Manageability Engine
-        ```
-
-    2. Build `vcpkg.exe` using the following command:
-
-        ``` bash
-        bootstrap-vcpkg.bat
-        ```
-        
-    3. Set up user-wide integration in order to allow Vcpkg to work with Visual Studio. 
-
-        ``` bash
-        vcpkg integrate install
-        ```
-
-    Build CPPRestSDK
-
-    1. Make sure you are still in the ../vcpkg directory.
-
-    2. Install the C++ REST SDK package using Vcpkg. C++ REST SDK is a library for cloud-based client-server communication with modern API design.  Find out more about it [here](https://github.com/microsoft/cpprestsdk).
-
-        !!! note
-            This will take 15–25 minutes, depending on download speeds.
-
-        ``` bash
-        vcpkg install cpprestsdk:x64-windows-static
-        ```
-    3. Navigate back to the `./rpc` directory
-
-        ``` bash
-        cd ..
-        ```
-
-=== "Linux"
-    1. To install the required dependencies; enter the following command:
+We leverage GitHub Actions as a means to build RPC automatically leveraging Github's CI/CD Infrastructure. This avoids having to deal with the challenges of getting your build environment just right on your local machine and allows you to get up and running much faster. However, if you wish to do this locally, please follow the instructions [here](./buildRPC.md).
 
-        ``` bash
-        sudo apt install git cmake build-essential libboost-system-dev libboost-thread-dev libboost-random-dev libboost-regex-dev  libboost-filesystem-dev libssl-dev zlib1g-dev
-        ```
+Read more about GitHub Actions [here](https://github.blog/2019-08-08-github-actions-now-supports-ci-cd/#:~:text=GitHub%20Actions%20is%20an%20API,every%20step%20along%20the%20way.)
 
-### Build RPC
-    
-1. Create a new directory called `build` within the `./rpc` directory and change to it.
+If you prefer to build manually on the command line, read more about how to build [here](../Microservices/RPC/buildRPC_Manual.md).
 
-    ``` bash
-    mkdir build && cd build
-    ```
+<img src="../../assets/animations/forkandbuild.gif" width="500"  />
 
-2. Run the following commands to build RPC.
+**To build the RPC with Github Actions:**
 
-    === "Windows"
+1. Create a fork of the repository.
 
-        Generate the CMake config
-        ``` bash
-        cmake .. -DVCPKG_TARGET_TRIPLET=x64-windows-static -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake
-        ```
+    [Fork rpc on github](https://github.com/open-amt-cloud-toolkit/rpc/fork){: .md-button .md-button--primary }
 
-        Build the RPC executable
-        ```bash
-        cmake --build . --config Debug
-        ```
+2. Click on **Actions**, the tab at the top, and select **Build RPC (Native) Debug/Release**.
 
-        Change to Debug directory
-        ``` bash
-        cd Debug
-        ```
+3. Click the **Run Workflow** dropdown, select **Branch:master**, and click the **Run Workflow** button.
```

Comment:
@rsdmike Do we want them to select **Branch:master** or should this be a tagged release?

Reply:
@rsdmike Hey, Mike, Any thoughts on Matt's point above.

Reply:
I have implemented a compromise to move forward.